R/visualize.R

```diff
@@ -0,0 +1,239 @@
+#' Fairness Accuracy Tradeoff Visualization
+#'
+#' @description
+#' These functions specialize in comparing the Fairness and Accuracy between learners through visualizations.
+#' From the visualization, users could see the tradeoff between fairness metrics and accuracy.
+#' Those insights could help the users to choose the optimum model from their model sets.
+#' And the fairness consistency of the learners by visualizing the tradeoff for different resampling iterations.
+#' It could take multiple types of inputs, like ([PredictionClassif]), ([BenchmarkResult]) and ([ResampleResult]).
+#'
+#'
+#' @param object ([PredictionClassif])\cr ([BenchmarkResult])\cr ([ResampleResult])\cr
+#' The binary class prediction object that will be evaluated. Only one data task is allowed for BenchmarkResult or ResampleResult
+#' * If provided a ([PredictionClassif]). Then only one point will indicate the accuracy and fairness metrics for the current predictions.
+#' * If provided a ([ResampleResult]). Then the plot will compare the accuracy and fairness metrics for the same model, but different resampling iterations.
+#' * If provided a ([BenchmarkResult]). Then the plot will compare the accuracy and fairness metrics for all models and all resampling iterations. Points are coloured acording to the models.
+#'
+#' @param ...
+#' Arguments to be passed to methods. Such as:
+#'
+#' * `fairness_measure` ([Measure])\cr
+#' The fairness measures that will evaluated on object, could be single measure (`msr`) or list of measures (`msrs`), check [mlr3::Measure]. Default measure set to be \code{msr("fairness.fpr")}
+#'
+#' * `accuracy_measure` ([Measure])\cr
+#' The accuracy measure that will evaluated on object, could be single measure \link{msr} or measures \link{msrs}. Default measure set to be \code{\link[mlr3:MeasureClassif]{msr("classif.acc")}}
+#'
+#' * `task` ([TaskClassif])\cr
```

Comment:
We expect Fairness Accuracy Tradeoff Visualization will only work for classification tasks. Because we cannot measure fairness metrics on classification tasks for now.